test/test_cli_integration.py

```diff
@@ -166,7 +176,64 @@ def test_https_cli_can_set_and_get_a_defined_variable(self):
             **self.HTTPS_CA_BUNDLE_ENV_VAR
         })
 
+        self.set_variable(CliIntegrationTest.DEFINED_VARIABLE2, "r")  # TODO: value should be special chars and it fails "!@#$%asdfgh&*()g,lpokmnjiub")
+        self.set_variable(CliIntegrationTest.DEFINED_VARIABLE3, "1")
+        output = self.get_variable([CliIntegrationTest.DEFINED_VARIABLE2, CliIntegrationTest.DEFINED_VARIABLE3])
         self.assert_set_and_get(CliIntegrationTest.DEFINED_VARIABLE_ID)
+        self.assert_json_equals(output,
+                                "{ \"" + CliIntegrationTest.DEFINED_VARIABLE2 + "\": \"r\", \"" + CliIntegrationTest.DEFINED_VARIABLE3 + "\": \"1\" }")
+
+    @integration_test
+    def test_https_cli_can_batch_get_multiple_variables(self):
```

Comment:
why is this in green? This already exists

Reply:
it was in the http not https or the opposite :)